docs/core/tools/dotnet-sln.md

```diff
@@ -60,17 +81,17 @@ dotnet sln add [-h|--help]
 
 - **`SOLUTION_FILE`**
 
-  The solution file to use. If not specified, the command searches the current directory for one. If there are multiple solution files in the directory, one must be specified.
+  The solution file to use. If not specified, the command searches the current directory for one. It fails if there are multiple solution files in the directory.
 
 - **`PROJECT_PATH`**
 
-  The path to the project to add to the solution. Add multiple projects by adding one after the other separated by spaces. Unix/Linux shell [globbing pattern](https://en.wikipedia.org/wiki/Glob_(programming)) expansions are processed correctly by the `dotnet sln` command.
+  The path to the project to add to the solution. Add multiple projects by adding one after the other separated by spaces. (See the fourth example.) Unix/Linux shell [globbing pattern](https://en.wikipedia.org/wiki/Glob_(programming)) expansions are processed correctly by the `dotnet sln` command.
```

Comment:
It might be best not to add "(See the fourth example.)".  It's likely to become incorrect if we add, remove, or reorder the examples.